connectors/sources/mssql.py

```diff
@@ -332,15 +340,20 @@ async def data_streamer(self, table=None, query=None):
         Yields:
             list: It will first yield the column names, then data in each row
         """
+        self._logger.debug(
+            f"Streaming records from database for table: {table} or using query: {query}"
+        )
```

Comment:
I'm not the biggest fan of using `or` inside a log statement as it adds ambiguity. I think we should have two distinct log messages depending on the value of `query`. You could also move the assignment of the parameter for `get_cursor` also into this condition to not check it twice.

Reply:
Sure, i've bifurcated this and made it dependant on the value of `query`.